packages/nx-plugin/src/plugin/target/targets.ts

```diff
@@ -0,0 +1,22 @@
+import { readdir } from 'node:fs/promises';
+import { CP_TARGET_NAME } from '../constants';
+import type { NormalizedCreateNodesContext } from '../types';
+import { createConfigurationTarget } from './configuration-target';
+
+export async function createTargets(
+  normalizedContext: NormalizedCreateNodesContext,
+) {
+  const { targetName = CP_TARGET_NAME } = normalizedContext.createOptions;
+  const rootFiles = await readdir(normalizedContext.projectRoot);
+  return rootFiles.some(filename =>
+    filename.match(/code-pushup\.config.(\w*\.)*(ts|js|mjs)$/),
```

Comment:
To ensure file name starts with `code-pushup...`.

```suggestion
    filename.match(/^code-pushup\.config.(\w*\.)*(ts|js|mjs)$/),
```